src/main/java/lotto_auto/model/LottoResult.java

```diff
@@ -0,0 +1,58 @@
+package lotto_auto.model;
+
+public enum LottoResult {
+
+    FIRST(true, 2000000000L, 1, 6),
```

Comment:
돈과 같은 1000단위로 구분은 다음과 같이 underbar(_)를 활용하면 코드의 가독성이 좋아질 수 있어요!
```suggestion
    FIRST(true, 2_000_000_000L, 1, 6),
```